liquibase-core/src/main/java/liquibase/util/ColumnOwnerType.java

```diff
@@ -0,0 +1,10 @@
+package liquibase.util;
+
+/**
+ * Used to indicate what type of relation owns a column.
+ */
+public enum ColumnOwnerType {
+    TABLE,
+    VIEW,
+    UNKNOWN
```

Comment:
I don't think we should have UNKNOWN as a type. These definitions exist for us to know what to do, and a type of UNKNOWN is not something we can know what to do with, so it just adds code and user complexity.

The variables of this enum type can be null, so better to rely on that single way to define "don't know the column type" than introduce this.

Reply:
Good point. I removed the "UNKNOWN" value.